configuration/pih/htmlforms/mentalHealthIntake.xml

```diff
@@ -211,9 +211,27 @@
         </style>
     </ifMode>
 
```

Comment:
Added the necessary javascript to support the commented out section below.  If uncommented, this should work in the same way as the NCD logic works on the ncd-adult-initial, just with a different concept set.